web3js-ext/src/web3/web3.ts

```diff
@@ -54,3 +85,90 @@ export class KlaytnWeb3 extends Web3 {
       }
   }
 }
+
+
+// Make OpenAPI-Generator ApiClient from ConnectionInfo
+function makeApiClient(url?: ConnectionInfo | string): any {
```

Comment:
In ethers-ext, we had ConnectionInfo from `wallet.provider`.
In web3js-ext, we have RequestInit from `web3.provider`.

Extract options from what we have here.
```suggestion
function makeApiClient(url: string, opts?: RequestInit)
```

Reply:
Yes. I agree.